src/svcutilcore/pkg/files/dotnet-svcutilcore.runtimeconfig.json

```diff
@@ -0,0 +1,9 @@
+{
+  "runtimeOptions": {
+    "tfm": "netcoreapp2.1",
+    "framework": {
+      "name": "Microsoft.NETCore.App",
+      "version": "2.0.0"
```

Comment:
I am actually not very sure if 2.0.0 MS.NC.App will work with tfm netcoreapp2.1. We probably should remove the framework section altogether or specify a particular version, for example, the version .NET Core just released for 2.1 preview 1.

Anyway, if this passes CI, let's still get this in and do a test run on VSTS. I'd like to unblock nightly run first.